data-dictionary.yaml

```diff
@@ -0,0 +1,110 @@
+title: mCODE Data Dictionary
+informationTabContent:
+  General notes: |-
+    **Profiles not shown in data dictionary:** Not all profiles used by mCODE are included in the Data Dictionary. mCODE uses FHIR profiles for Blood Pressure, Height, Weight, and the US Core Laboratory Result Observation Profile for Complete Blood Count (CBC), and Comprehensive Metabolic Panel (CMP). These profiles are not included in the Data Dictionary because they are defined externally to mCODE.
+
+    **Data elements not shown in data dictionary:** Only elements marked as "Must Support" in the FHIR Implementation Guide are included in this data dictionary. These are elements that implementers are expected to provide meaningful support for; see https://build.fhir.org/ig/HL7/fhir-mCODE-ig/branches/master/conformance.html#mustsupport-interpretation for more information.
```

Comment:
You shouldn't cite build.fhir.org continuous integration build. Refer to the published version -- or the page it will be on when this version is published.

Reply:
I changed this to `http://hl7.org/fhir/us/mcode/conformance.html#mustsupport-interpretation`